csuntechlab/iSTART
# STR-188 -- FE: Implemented card-flip state and checks

# Description
Created functionality to save cardflip states.
  
# Testing
1. Login through the Module route.
2. Visit a card-flip slide.
3. Ensure when clicking through all the cards (*don't hit continue yet*) it enables the continue button.
4. Click the **Back** button and click **Continue**; ensure it still allows you to continue and, all the cards are flipped to its original state.
5. Un-click a card and check to see if the continue button is still enabled.
6. Click the **Back** button and click **Continue**; ensure the continue button is enabled even if you have a card un-flipped. (*as long as they clicked all the cards one time*)

# Installation
'Are there any dependencies the reviewer needs to download?'
